functions/Set-DbaDatabaseOwner.ps1

```diff
@@ -107,24 +107,21 @@ Sets database owner to 'sa' on the db1 and db2 databases if their current owner
 			#Otherwise, use all databases on the instance where owner not equal to -TargetLogin
 			Write-Verbose "Gathering databases to update"
 			
+            #use online/available dbs where owner and target login do not match
+            #exclude system dbs
+            $dbs = $server.Databases | Where-Object {$_.Status -eq 'Normal' -and $_.Owner -ne $TargetLogin -and @('master', 'model', 'msdb', 'tempdb', 'distribution') -notcontains $_.Name}
```

Comment:
So would we prefer a warning/error if the database can't be updated instead of filtering out offline/restoring/suspect databases?